File: fizzbuzz.py
# Fizzbuzz
#
# Prints numbers from 1 to 100. If the number is divisible by 3, 'Fizz'
# is printed instead. If the number is divisible by 5, 'Buzz' is
# printed instead. If the number is divisible by both 3 and 5,
# 'Fizzbuzz' is printed instead.

for number in xrange(1, 101):
    if number % 3 == 0 and number % 5 == 0:
        print "Fizzbuzz"
    elif number % 3 == 0:
        print "Fizz"
    elif number % 5 == 0:
        print "Buzz"
    else:
        print number

File: c/fizzbuzz.c
/* Fizzbuzz:
 * Print numbers 1-100. If divisible by three, print "Fizz". If divisible
 * by 5, print "Buzz". If difisible by both 3 and 5, print "Fizzbuzz". */

#include <stdio.h>

int main(void)
{
    int i;
    for (i=0; i<=100; i++)
    {
        if (i % 3 == 0 && i % 5 == 0)
        {
            printf("Fizzbuzz\n");
        }
        else if (i % 3 == 0)
        {
            printf("Fizz\n");
        }
        else if (i % 5 == 0)
        {
            printf("Buzz\n");
        }
        else
        {
            printf("%d\n", i);
        }
    }
    return 0;
}
